### ISSUE TITLE
BKB / Laguna Blade

### ISSUE THREAD

**Issue by Azzurite** [kixelated/mechquiz#1]
Laguna Blade with Aghs goes through, but without not. But there's no way to know if Laguna Blade is Aghs upgraded or not.

**Comment by sarmad3000** [comment]
Agreed with this one, the quiz instructions need to clarify about Aghs upgrades.
